src/com/kainos/tdp/employeemanager/Menu.java

```diff
@@ -0,0 +1,23 @@
+package com.kainos.tdp.employeemanager;
+
+public class Menu {
+	
+	public void printMenu() {
+		System.out.println("******************************");
+		System.out.println("       Welcome to the         ");
+		System.out.println("     Employee Management      ");
+		System.out.println("           System             \n");
+		System.out.println(" 1. Insert a New Employee     \n");
+		System.out.println(" 2. View Employee Details     \n");
+		System.out.println("******************************");
+	}
+	
+	public boolean handleUserChoice(int userInput) {
```

Comment:
This is duplicate code. Make the Main.java one static then call `Main.checkUserChoiceValid()` from this class.